src/renderer/services/deposit/options.ts

```diff
@@ -0,0 +1,119 @@
+import {number64} from "@chainsafe/eth2.0-types";
+
+/**
+ * Defines properties of deposit contract.
+ */
+export interface IEth1Options {
+    depositContract: {
+        deployedAt: number64;
+        address: string;
+        abi: (string | any)[] | string;
+        bytecode: string;
+    };
+}
```

Comment:
Adjust this to your use case. You only need address and abi as options and rename this to IDepositContract